src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/css/app.css

```
@@ -80,17 +84,18 @@ a, .btn-link {
     }
 
         .loading-progress circle:last-child {
-            stroke: var(--blazor-load-percentage-color);
+            stroke: #1b6ec2;
             stroke-dasharray: calc(3.141 * var(--blazor-load-percentage) * 0.8), 500%;
```

Sorry, I didn't notice before the scenario where the value isn't yet set. There is a native way to handle this in CSS. It's like this:

```suggestion
            stroke-dasharray: calc(3.141 * var(--blazor-load-percentage, 0%) * 0.8), 500%;
```